eoscostarica/guide.eoscostarica.io
# Fix/dark theme images

### GH Issue

Resolves #229  

### Steps to test
1. Check rendering images are working correctly in both mode
1. Check cards are rendering correctly in dark mode

#### CheckList
